hta/utils/utils.py

```diff
@@ -174,3 +178,21 @@ def flatten_column_names(df: pd.DataFrame) -> None:
     """Flatten a DataFrame's a multiple index column names to a single string"""
     if isinstance(df.columns, pd.MultiIndex):
         df.columns = ["_".join(col).rstrip("_") for col in df.columns]
+
+
+def get_mp_pool_size(obj: object, num_objs: int) -> int:
+    """
+    Estimate the maximum pool size for multiprocessing
+
+    Args:
+        obj (object): one instance of the objects to be processed
+        num_objs (int): the total number of objects to be processed
+
+    Return:
+        the recommend pool size
```

Comment:
please add the return type in the return section. HTA follows this style in the doc strings
https://github.com/facebookresearch/HolisticTraceAnalysis/blob/main/hta/trace_diff.py#L83-L100